src/libs/svg/lv_svg_render.c

```diff
@@ -1567,23 +1594,32 @@ static void _render_text(const lv_svg_render_obj_t * obj, lv_vector_dsc_t * dsc,
         lv_matrix_multiply(&dsc->current_dsc.matrix, matrix);
     }
 
+    bool build_path = false;
     if(lv_array_size(&text->path->ops) == 0) { /* empty path */
-        lv_vector_path_t * glyph_path = lv_vector_path_create(LV_VECTOR_PATH_QUALITY_MEDIUM);
-        // draw text contents and spans
-        lv_matrix_t mtx;
-        lv_matrix_identity(&mtx);
-        lv_matrix_translate(&mtx, text->x, text->y);
-        for(uint32_t i = 0; i < lv_array_size(&text->contents); i++) {
-            lv_svg_render_obj_t * ptext = *((lv_svg_render_obj_t **)lv_array_at(&text->contents, i));
-            lv_svg_render_content_t * content = (lv_svg_render_content_t *)ptext;
-
-            if(content->render_content) {
-                content->render_content(content, dsc, &mtx);
-            }
-            else {
+        build_path = true;
+    }
+
+    // draw text contents and spans
```

Comment:
```suggestion
    /* draw text contents and spans */
```
Maybe on other places too.

Reply:
Does the code_format  script solve this kind of thing?

Reply:
Nope.